public/script.js

```diff
@@ -1,5 +1,44 @@
-const earthDateParagraph = document.querySelector("#earth-date");
 const earthDate = new Date();
-const dateText = document.createTextNode(earthDate.toLocaleString());
 
-earthDateParagraph.appendChild(dateText);
+function displayCurrentEarthDate() {
+  const earthDateParagraph = document.querySelector("#earth-date");
+  const dateText = document.createTextNode(earthDate.toLocaleString());
+
+  earthDateParagraph.appendChild(dateText);
```

Comment:
One thing that could make it a little bit lighter on the mental overhead is to chain together the `querySelector` and the `appendChild`.

Chaining things together (...within reason) in general can make it a little lower on the mental overhead as it reduces the number of variables floating about.